Horion/Command/Commands/HelpCommand.cpp

```diff
@@ -2,22 +2,19 @@
 
 #include "../CommandMgr.h"
 
-HelpCommand::HelpCommand() : IMCCommand("help", "Lists all commands", "")
-{
+HelpCommand::HelpCommand() : IMCCommand("help", "Lists all commands", "") {
 	registerAlias("h");
 }
 
 
-HelpCommand::~HelpCommand()
-{
+HelpCommand::~HelpCommand() {
 }
 
-bool HelpCommand::execute(std::vector<std::string>* args)
-{
+bool HelpCommand::execute(std::vector<std::string>* args) {
 	g_Data.getGuiData()->displayClientMessageF("==========");
 	std::vector<IMCCommand*>* commandList = cmdMgr->getCommandList();
 	for (auto it = commandList->begin(); it != commandList->end(); ++it) {
-		IMCCommand* plump = *it;
+		IMCCommand* plump = *it; // TODO: change plump
```

Comment:
just change it to something like "command" or "cmd", idk

Reply:
gotcha richard 😳